src/Raven.Server/Documents/Patch/V8/BlittableObjectInstanceV8.cs

```diff
@@ -0,0 +1,1046 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using V8.Net;
+using Lucene.Net.Store;
+using Raven.Client;
+using Raven.Client.Documents.Indexes;
+using Raven.Server.Documents.Indexes;
+using Raven.Server.Documents.Indexes.Static;
+using Raven.Server.Documents.Queries.Results;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Sparrow.Server.Json.Sync;
+using Raven.Server.Utils;
+
+namespace Raven.Server.Documents.Patch.V8
+{
+    [DebuggerDisplay("_blittable JS object")]
+    public class BlittableObjectInstanceV8 : IBlittableObjectInstance
+#if DEBUG
+    , IV8DebugInfo
+#endif
+    {
+        public class CustomBinder : ObjectBinderEx<BlittableObjectInstanceV8>
+        {
+            public CustomBinder() : base()
```

Comment:
Due to V8.Net design: 'ObjectBinder' based instance is associated with the specified object instance. ObjectBinder implements ManagedObject interface by mediating with the bound object. We may have a generic binder for a given class (automatically scans and filters properties and methods) or implement it manually (like here and for Counter, TimeSeries, ...). 
BlittableObjectInstance has two constructors, plays his own role and couldn't be a binder itself. But it seems that others, inheriting from ObjectInstanceBase like CounterEntryObjectInstance, could be binders itself (for DynamicCounterEntry class in case of CounterEntryObjectInstance). I set todo for this refactoring (after we get the working version).